bliss/conf/base_config.yaml

```diff
@@ -81,7 +81,8 @@ encoder:
     input_transform_params:
         use_deconv_channel: false
         concat_psf_params: false
-        log_transform: true
+        log_transform:
+            scales: [10]  # add new log transforms
```

Comment:
@XinyueLi1012 I think the ability to preprocess the images with multiple log transformations simply by modifying the config file will be useful for cataloging DC2.